packages/flame_riverpod/lib/src/component_ref.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+/// A wrapper around [WidgetRef]. Methods that correspond to interactions with
+/// the widget tree, such as [WidgetRef.watch] are not exposed.
+class ComponentRef {
+  ComponentRef(this._widgetRef);
+  final WidgetRef _widgetRef;
+
+  BuildContext get context => _widgetRef.context;
```

Comment:
What is this context? In Flutter, every widget gets its own build context, so which build context is this?

Reply:
Some widget that extends ConsumerWidget or ConsumerStatefulWidget provides its context to a FlameGame.